src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -555,15 +556,27 @@ public function prependExtensionConfig($name, array $config)
      *  * The parameter bag is frozen;
      *  * Extension loading is disabled.
      */
-    public function compile()
+    public function compile(/* $resolveEnvPlaceholders = false */)
```

Comment:
How about using a flag (e.g. `self::RESOLVE_ENV_PLACEHOLDERS`) instead of a boolean, so that addition features may be introduced without requiring additional parameters?